README.md

```diff
@@ -109,6 +109,8 @@ Read more about how to install the example webhook [here](deploy/kubernetes/webh
 
 * `--port`: Secure port that the webhook listens on (default 443)
 
+* `--kubeconfig <path>`: Path to Kubernetes client configuration that the webhook uses to connect to Kubernetes API server. When omitted, default token provided by Kubernetes will be used. This option is useful only when the snapshot controller does not run as a Kubernetes pod, e.g. for debugging.
```

Comment:
Please add also a sentence or two to "Validating Webhook" chapter above that snapshot classes are validated too and how it work (only one class can be default for each CSI driver).

Reply:
Update, let me know if the new section makes sense?

Reply:
makes sense, thanks!